liquibase-core/src/main/java/liquibase/change/AbstractSQLChange.java

```diff
@@ -282,7 +282,6 @@ protected String normalizeLineEndings(String string) {
     }
 
     public static class NormalizingStream extends InputStream {
-        private ByteArrayInputStream headerStream;
         private PushbackInputStream stream;
```

Comment:
This probably doesn't have to be a PUshbackInputSTream anymore and can just be the InputStream itself?

Reply:
Updated to InputStream.